EIPS/eip-draft_backward_compat_PQ_migration.md

```diff
@@ -0,0 +1,198 @@
+
+  The Test Cases section should include expected input/output pairs, but may include a succinct set of executable tests. It should not include project build files. No new requirements may be be introduced here (meaning an implementation following only the Specification section should pass all tests here.)
+  If the test suite is too large to reasonably be included inline, then consider adding it as one or more files in `../assets/eip-####/`. External links will not be allowed
+
+  TODO: Remove this comment before submitting
+-->
+
+## Reference Implementation
+
+<!--
+  This section is optional.
+
+  The Reference Implementation section should include a minimal implementation that assists in understanding or implementing this specification. It should not include project build files. The reference implementation is not a replacement for the Specification section, and the proposal should still be understandable without it.
+  If the reference implementation is too large to reasonably be included inline, then consider adding it as one or more files in `../assets/eip-####/`. External links will not be allowed.
+
+  TODO: Remove this comment before submitting
+-->
+
+## Security Considerations
+
+We aim to minimally achieve the Quantum Level 1 Security as defined by [NIST](https://csrc.nist.gov/csrc/media/Presentations/2022/the-beginning-of-the-end-the-first-nist-pqc-standa/images-media/pkc2022-march2022-moody.pdf). This requires the security to be at least equivalent to AES-128.
+
+For MPCitH zero-knowledge proof, we propose the use of [Katz-Kolesnikov-Wang](https://dl.acm.org/doi/pdf/10.1145/3243734.3243805) with parameters $`N=16`$, $`M=250`$, $`\tau=36`$ to achieve NIST Quantum Level 1 security.
+
+Since zkSTARK computations do not happen over a finite field (such as zkSNARKS), it is generally accepted that zkSTARK proofs are quantum-safe, as long as the hash used primitive is collapsing and is large enough, e.g., 256-bit key strength, to prevent a brute-force attack using Grover's algorithm.
+
+Our design has not yet taken into account how it affects Multi-Party Computation (MPC) wallets.
+
+## Disclosure
+
+The proposed design outlined in this EIP incorporates certain features that may be subject to claims in pQCee's patent-pending application (PCT [WO2023080842](https://patentscope.wipo.int/search/en/detail.jsf?docId=WO2023080842)). While it is important to ensure that adoption of the proposed post-quantum signature scheme remains accessible to all Ethereum stakeholders, this has to be balanced against the need to recognize and acknowledge the intellectual property rights associated with its development. We recognize the value of fostering an open and collaborative ecosystem that promotes widespread adoption and innovation within the Ethereum community, and as such, we welcome constructive dialogue with interested parties to explore appropriate licensing arrangements of the patented technologies to achieve these objectives.
```

Comment:
EIPs are meant to be open for anyone to implement. I do very much appreciate this disclosure, but I don't believe we can accept a proposal unless it also includes an irrevocable patent grant.

Reply:
Thank you for the feedback. This current design is preliminary. We will further engage the community to get more feedback regarding this.

Reply:
marking this unresolved for @SamWilsn  to take a call on this